### PR TITLE
Styling tweaks and code cleanup

### PR DESCRIPTION
**Pull request by nickbutcher**

### CODE REVIEW

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt [hunk 1]
```diff
@@ -209,9 +211,13 @@ fun ErrorSnackbar(
...
                         onErrorAction()
                         onDismiss()
-                    }
+                    },
+                    contentColor = contentColor()
                 ) {
-                    Text("RETRY")
+                    Text(
+                        text = "RETRY",
+                        color = Red300
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt:219]: Code smell using a hardcoded color?

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt [hunk 2]
```diff
@@ -46,7 +45,9 @@ import com.example.jetnews.ui.navigateTo
 
 @Composable
 fun PostCardPopular(post: Post, modifier: Modifier = Modifier) {
-    Card(modifier = modifier.preferredSize(280.dp, 240.dp), shape = RoundedCornerShape(4.dp)) {
+    Card(
+        shape = MaterialTheme.shapes.medium,
+        modifier = modifier.preferredSize(280.dp, 240.dp)) {
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt:50]: Really need to use a hardcoded size?

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt (hunk 1 above)
Comment (nickbutcher) [reply JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt:219]: Yeah, design calls for non-theme values (b/155218328). Snackbars are weird. Will fix.

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt (hunk 2 above)
Comment (nickbutcher) [reply JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardYourNetwork.kt:50]: 🤷‍♂️

**Pull request merged**